src/Gdi32.Desktop/Gdi32+StockObjects.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="StockObjects"/> nested type.
+    /// </content>
+    public static partial class Gdi32
+    {
+        public enum StockObjects
```

Comment:
.NET Framework guidelines suggest using singular nouns for enum types unless they are `[Flags]` enums, which this is not. Please rename to `StockObject`.